lib/block-supports/index.php

```diff
@@ -89,11 +89,11 @@ function gutenberg_apply_block_supports( $block_content, $block ) {
 
 	// Apply new styles and classes.
 	if ( ! empty( $new_classes ) ) {
-		$block_root->setAttribute( 'class', esc_attr( implode( ' ', $new_classes ) ) );
+		$block_root->setAttribute( 'class', implode( ' ', $new_classes ) );
```

Comment:
Good thought on the comment @Addison-Stavlo, how about this?

```suggestion
		// `DOMElement::setAttribute` handles attribute value escaping.
		$block_root->setAttribute( 'class', implode( ' ', $new_classes ) );
```

Reply:
That works for me.  Thanks!